types/mongoose/index.d.ts

```diff
@@ -7,6 +7,7 @@
 //                 jendrikw <https://github.com/jendrikw>
 //                 Ethan Resnick <https://github.com/ethanresnick>
 //                 vologa <https://github.com/vologab>
+//                 jussikinnula <https://github.com/jussikinnula>
```

Comment:
Do you want to help maintain the mongoose type definitions and be notified of every PR? If not, remove this line.

Reply:
I did added myself just because of that, since MongoDB 4.0 upgrade will need more work for Mongoose (and also the related typing packages), so I want to give a helping hand on reviewing the changes. And most probably I might be doing the updates myself as well. Thanks for asking! :-)

Reply:
Thank you for your support!